tests/unit_tests/test_cli.py

```diff
@@ -91,8 +95,8 @@ def test_get_plans(runner: CliRunner):
     )
 
     plans = runner.invoke(main, ["controller", "plans"])
-    assert response.call_count == 1
     assert plans.output == "my-plan\n    Args\n      id=string (Required)\n"
+    assert response.call_count == 1
```

Comment:
```suggestion
    assert response.call_count == 1
    assert plans.output == "my-plan\n    Args\n      id=string (Required)\n"
```

Strange re-ordering of these lines